src/main/java/frc/robot/subsystems/vision/VisionEstimation.java

```diff
@@ -110,6 +121,10 @@ public void periodic() {
         // getAtomicCountBlinkyTelemetry.periodic();
         getAtomicCountPinkyTelemetry.periodic();
         getAtomicCountClydeTelemetry.periodic();
+        // confidenceInkyTelemetry.periodic();
+        // confidenceBlinkyTelemetry.periodic();
+        confidencePinkyTelemetry.periodic();
+        confidenceClydeTelemetry.periodic();
```

Comment:
The `periodic()` function of these helper telemetry classes will send the data to SmartDashboard once every sec or half sec (the reporting period is specified in their c'tor on lines 89-101).